docs/theme/tokens/application/editor.ts

```diff
@@ -0,0 +1,164 @@
+export const editor = {
+  background: { value: '{color.background.primary.value}' },
```

Comment:
I wonder if we'd be setting a better example by using the DTCG format and using $value and then just putting the $type at the very very top group level, so it only needs to be set once (since all of the tokens here are color tokens it seems?).